PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.kt

```diff
@@ -123,14 +123,16 @@ open class SparkSubmissionContentPanel(private val myProject: Project, val type:
         }
     }
 
+    open val clusterHint = "Spark clusters(Linux only)"
+
     open fun getErrorMessageClusterNameNull(isSignedIn: Boolean): String {
         return when {
             isSignedIn -> "Cluster name should not be null, please choose one for submission"
             else -> "Can't list cluster, please login within Azure Explorer (View -> Tool Windows -> Azure Explorer) and refresh"
         }
     }
 
-    val clustersSelectionPrompt: JLabel = JLabel("Spark clusters(Linux only)").apply {
+    val clustersSelectionPrompt: JLabel = JLabel("$clusterHint").apply {
```

Comment:
`val clustersSelectionPrompt: JLabel = JLabel(clusterHint).apply` might be clear. But never mind, no big deal.